amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/activity/preview/AmountSerializer.java

```diff
@@ -0,0 +1,25 @@
+package org.digijava.kernel.ampapi.endpoints.activity.preview;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+
+import org.codehaus.jackson.JsonGenerator;
+import org.codehaus.jackson.JsonProcessingException;
+import org.codehaus.jackson.map.JsonSerializer;
+import org.codehaus.jackson.map.SerializerProvider;
+
+/**
+ * Amount json serializer used for activity preview endpoint
+ * 
+ * @author Viorel Chihai
+ *
+ */
+public class AmountSerializer extends JsonSerializer<Double>  {
+
+    @Override
+    public void serialize(Double value, JsonGenerator jgen, SerializerProvider provider)
+            throws IOException, JsonProcessingException {
+        jgen.writeString(BigDecimal.valueOf(value).setScale(0, BigDecimal.ROUND_HALF_UP).toString());        
```

Comment:
I checked for usages and I'd like to confirm: should the amounts like PPC/RPC, subtotals be rounded up to the decimal point in preview? I understand normally we don't provide decimal point entries in the orig. currency and it may be reasonable to do the same for converted amounts, however AMP uses amount formats that allows decimal points and for values like `undisbursed_balance` it may make sense, so just want to be sure this rounding is used correctly.